pyproject.toml

```diff
@@ -1,2 +1,6 @@
 [build-system]
 requires = ["setuptools", "wheel", "setuptools-rust"]
+
+[tool.black]
+line-length = 100
```

Comment:
I don't feel too strongly about this, but I have the rustfmt settings at 80 characters because of my personal preference (I do most of my dev in 80 character wide terminals). If we want to go for 100 characters here we probably should update the rustfmt config in a follow up PR.